Modules/Sources/AsyncImageKit/ImagePrefetcher.swift

```diff
@@ -89,6 +89,7 @@ public final class ImagePrefetcher {
                 value.task?.cancel()
             }
             queue.removeAll()
+            numberOfActiveTasks = 0
```

Comment:
I believe `numberOfActiveTasks` needs to be updated in `stopPrefetching` too.

Reply:
`stopPrefetching` cancel the task `task.task?.cancel()` but doesn't manually manipulate the `queue`. The task will then need to cancel or run until the completion depending what's state it's in. So updating `numberOfActiveTasks` is not needed there. This is the part I did test (manually). I didn't test `stopAll` though because it's not used anywhere.

Reply:
Oh. I see it now. `numberOfActiveTasks` is updated in the `actuallyPrefetchImage` task when the task is cancelled.